lib/bundler/source/rubygems.rb

```diff
@@ -302,6 +302,7 @@ def remote_specs
           # the gemspecs of those gems, if the non-api sites contain more than
           # about 100 gems, we just treat all sites as non-api for speed.
           allow_api = idx.size < API_REQUEST_LIMIT && dependency_names.size < API_REQUEST_LIMIT
+          Bundler.ui.debug "Need to query more then #{API_REQUEST_LIMIT} gems" unless allow_api
```

Comment:
Sure.
My first version included such info, but I've removed it since the current output will be:

```
$ DEBUG=1 bundle
HTTP GET https://bundler.rubygems.org/api/v1/dependencies
HTTP 200 OK
Need to query more then 100 gems
Fetching source index from https://rubygems.org/
```

Do you think the following is better?:
```
$ DEBUG=1 bundle
HTTP GET https://bundler.rubygems.org/api/v1/dependencies
HTTP 200 OK
Need to query more then 100 gems. Downloading full index instead...
Fetching source index from https://rubygems.org/
```